libs/librtlnumber/verify_librtlnumber.sh

```diff
@@ -98,7 +98,7 @@ for INPUT in ${0%/*}/regression_tests/*.csv; do
 		then
 			echo "--- PASSED == $TEST_LABEL ( ${OUTPUT_AND_RESULT} ) "
 
-		elif [ "pass" == "$(${0%/*}/rtl_number is_true $(${0%/*}/rtl_number ${OUTPUT_AND_RESULT} == ${EXPECTED_RESULT}))" ]
+		elif [ "pass" == "$(${0%/*}/rtl_number is_true $(${0%/*}/rtl_number ${OUTPUT_AND_RESULT} === ${EXPECTED_RESULT}))" ]
```

Comment:
I think at this time, we would be better served with just comparing the output of the === operation.
You could, in turn, take out the is_true (originally temporary) that is only used for the tests.

Reply:
Agreed! :smiley:

Reply:
Surely you need to test is_true? I assume that is what you use to evaluate ternary conditionals?

Reply:
```                                
VNumber eval = V_CASE_EQUAL(*(child_condition->types.vnumber), *(item->children[0]->types.vnumber));
if (V_TRUE(eval)) 
```
In CASE statements as well as IF and IF_Q

Reply:
It does not hurt to keep it so may as well leave it, but I was talking about `is_true` in the driver for the test harness, I don't know that we use this any more:
https://github.com/verilog-to-routing/vtr-verilog-to-routing/blob/master/libs/librtlnumber/main.cpp#L147-L155